### PR TITLE
chore: remove react native elements from examples

### PR DESCRIPTION
**Pull request by KiwiKilian**
## Description

I've removed React Native Elements from the examples. This also removes a lot of other dependency bloat and should speed up development. As a benefit, the app looks much nicer.

Bildschirmfoto 2024-10-22 um 15 38 13

Bildschirmfoto 2024-10-22 um 15 35 17

### CODE REVIEW

File: packages/examples/src/assets/index.d.ts [hunk 1]
```diff
@@ -7,8 +7,3 @@ declare module "*.png" {
   const content: number;
   export default content;
 }
-
-declare module "*.json" {
-  const content: string;
```
Comment (KiwiKilian) [inline packages/examples/src/assets/index.d.ts:12]: Previous type is wrong – the imports actually return the JSON object.

File: packages/examples/src/examples/FillRasterLayer/WatercolorRasterTiles.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const OPTIONS = [0, 0.25, 0.5, 0.75, 1];
+const DEFAULT_OPTION = 4;
+
+export default function WatercolorRasterTiles() {
+  const [value, setValue] = useState(OPTIONS[DEFAULT_OPTION]);
+
+  const rasterSourceProps = {
+    id: "stamenWatercolorSource",
+    tileUrlTemplates: [
+      "https://stamen-tiles.a.ssl.fastly.net/watercolor/{z}/{x}/{y}.jpg",
```
Comment (KiwiKilian) [inline packages/examples/src/examples/FillRasterLayer/WatercolorRasterTiles.tsx:17]: This is not available and we need an alternative

**Review by caspg — APPROVED** [review]
That's really epic job. Thanks!

**Comment by tyrauber** [comment]
Fantastic! Thank you.

**Pull request merged**